README.md

```diff
@@ -67,13 +67,41 @@ is up and running!
 ### Setting up the client
 
 The easiest way to interact with Deployment Manager is through the `dm` tool
-hitting a `kubectl` proxy. To set that up:
+hitting a `kubectl` proxy.
 
-1. Build the tool by running `make` in the deployment-manager repository.
-1. Run `kubectl proxy --port=8001 --namespace=dm &` to start a proxy that lets you interact
-with the Kubernetes API server through port 8001 on localhost. `dm` uses
+#### Creating a proxy
+
+You can run the following to start a proxy that lets you interact with the
+Kubernetes API server through port 8001 on `localhost`
+
+```
+kubectl proxy --port=8001 --namespace=dm &
+```
+
+`dm` will use
 `http://localhost:8001/api/v1/proxy/namespaces/dm/services/manager-service:manager`
-as the default service address for DM.
+as the default service address for the DM service.
+
+#### Getting the client
+You can get access to the client in one of two ways:
+
+1. Build the client by running `make` in the deployment-manager repository.
+1. Use the client from the container `gcr.io/dm-k8s-testing/dm`.
+
+**NOTE**: If you are using the client from the docker container, you will need
+to substitute the following docker command in place of `dm` in all of the
+examples below, or alias the command appropriately.
+
+```
+alias dm="docker run --net=host gcr.io/dm-k8s-testing/dm"
```

Comment:
Per the comment from @vaikas-google, the user may need to do more than what's being prescribed here to use dm successfully within a container.

Specifically, if they are deploying a configuration or a local template, not a template from a registry, then they will need a way to get the configuration or local template, plus any local input files, into the container.

One way to do that is to mount the folder containing the files as a volume, like this:

```
alias dm="docker run --net=host -v ./examples:/examples gcr.io/dm-k8s-testing/dm
```

Then, the file references will need to be relative to the path where the volume is mounted, in this case `/examples`. Using that mount, the sample command line given next in the README.md would become:

```
dm deploy /examples/guestbook/guestbook.yaml
```

Note that the path now starts at `/examples` instead of `examples`.

The other way to do that is to pass the files through stdin, which is mapped by default to the stdin of the command running in the container. However, we haven't yet implemented support for reading files from stdin, so that isn't yet an option. That said, I think we should document it as an option, and then go ahead an implement reading from stdin, so that the documentation provides a complete description of the intended behavior.